src/listr/tasks/tenderdash/initTaskFactory.js

```diff
@@ -0,0 +1,61 @@
+const { Listr } = require('listr2');
```

Comment:
The file name should be equal to a function or a class that it contains.

Reply:
Right, so:

- `platform/initTaskFactory.js` contains `platformInitTaskFactory`
- `tenderdash/initTaskFactory.js` contains `tenderdashInitTaskFactory`

Should I rename the functions or the files? Or move everything to another directory called `init` or something?

Reply:
File names atm are not equal to functions which they consist. It should be `tenderdashInitTask`